From 442c60f800173341e149d303f7c4b5c033f31a53 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 4 Apr 2011 22:46:38 -0500
Subject: do-not-initialize-terminal-twice.diff
 * The terminal should no longer be initialized twice.
   Patch: do-not-initialize-terminal-twice.diff
   Provided-by: Courtney Bane <debian-bugs-5265@cbane.org>
   Date: Thu, 07 Oct 2010 12:58:15 -0500
   Added-by: Rob Browning <rlb@defaultvalue.org>
   Status: incorporated upstream

  The Debian patch is taken from this upstream commit:

  revno: 100029
  author: Anonymous <emacs-bugs-7626@cbane.org>
  committer: Stefan Monnier <monnier@iro.umontreal.ca>
  branch nick: emacs-23
  timestamp: Sat 2010-09-11 14:53:21 +0200
  message:
    * src/term.c (tty_set_terminal_modes): Don't initialize twice (bug#7002).
---
 src/ChangeLog |    4 ++++
 src/term.c    |    1 -
 2 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/src/ChangeLog b/src/ChangeLog
index 59eaf48..8d42a1e 100644
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,3 +1,7 @@
+2010-09-11  Anonymous  <emacs-bugs-7626@cbane.org>  (tiny change)
+
+	* term.c (tty_set_terminal_modes): Don't initialize twice (bug#7002).
+
 2010-08-17  Andreas Schwab  <schwab@linux-m68k.org>
 
 	* eval.c (Flet, Feval, Fapply, apply_lambda): Use SAFE_ALLOCA_LISP
diff --git a/src/term.c b/src/term.c
index 7adaeea..4a8efcb 100644
--- a/src/term.c
+++ b/src/term.c
@@ -250,7 +250,6 @@ tty_set_terminal_modes (struct terminal *terminal)
             cmputc ('\n');
         }
 
-      OUTPUT_IF (tty, tty->TS_termcap_modes);
       OUTPUT_IF (tty, visible_cursor ? tty->TS_cursor_visible : tty->TS_cursor_normal);
       OUTPUT_IF (tty, tty->TS_keypad_mode);
       losecursor (tty);
-- 
1.7.4.1

